src/test/suite/gather.unit.test.ts

```diff
@@ -95,7 +95,7 @@ suite("code gathering unit tests", () => {
       }
 
       const cell: IVscCell = codeCells[codeCells.length - 1];
-      const program = gatherProvider.gatherCode(cell);
+      const program = await gatherProvider.gatherCode(cell);
```

Comment:
I tried to run the tests and found the tests are expecting `program` to be a `string`, hence added the `await`, but then found that `gatherCode` doesn't return anything. I.e. tests don't work. 
Either way just left these changes so they compile